FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.JobManagement/AzureStorage.cs

```diff
@@ -0,0 +1,42 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Fhir.Synapse.Common.Configurations;
+
+namespace Microsoft.Health.Fhir.Synapse.JobManagement
+{
+    public class AzureStorage : IStorage
+    {
+        public AzureStorage(IOptions<JobConfiguration> config)
```

Comment:
This is more like configurtion?

Reply:
Removed, the config values are parsed in `AzureStorageClientFactory` now.